stable/rocketchat/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+name: rocketchat
+version: 0.0.1
+description: Prepare to take off with the ultimate chat platform, experience the next level of team communications
+keywords:
+- chat
+- communication
+- http
+- web
+- application
+- nodejs
+- javascript
+- meteor
+home: https://rocket.chat/
+icon: https://cdn-www.rocket.chat/images/logo/logo.svg
+sources:
+- https://github.com/RocketChat/Docker.Official.Image/
+maintainers:
+- name: RocketChat
```

Comment:
@unguiculus I notice many of the bitnami created ones refer to their org name on github and add an email address.  But i've gone ahead and added two additional maintainers by github username.